stripe/tslint.json

```diff
@@ -0,0 +1,9 @@
+{
+    "extends": "../tslint.json",
+    "rules": {
+        // The Stripe client side APIs are not made available to package managers for direct installation.
+        // As explained compliance reasons. Source: https://github.com/stripe/stripe-node/blob/master/README.md#these-are-serverside-bindings-only
+        // A release date versioning schema is used to version these APIs. It is not compatable with semver.
+        "dt-header": false
```

Comment:
You could just use `0.0` in the header and include a comment somewhere in the source.